ironfish/src/webApi.ts

```diff
@@ -124,6 +124,16 @@ export class WebApi {
     return response.data.address
   }
 
+  async getMinAndMaxDepositSize(): Promise<{ minDepositSize: number; maxDepositSize: number }> {
+    const response = await axios.get<{ min_deposit_size: number; max_deposit_size: number }>(
```

Comment:
Before merging, we should at minimum test this endpoint.

Reply:
Endpoint is in staging: https://ironfish-api-staging.herokuapp.com/deposits/min_and_max_deposit_size

Reply:
👍